documentation/running_bolt_network.md

```diff
@@ -0,0 +1,92 @@
+# Run Bolt on Network Devices
+
+You can run Bolt commands that target network devices in order to configure, provision, and make
+other changes to those devices. However, interacting with network devices through Bolt can be
+different than interacting with other types of targets. These are some use cases and workflows that
+are useful when running against network devices.
+
+## Run multiline commands
+
+Files can't be written on certain devices, which prevents users from using `bolt script run` or
+`Invoke-BoltScript` to run multiline commands. With Bolt you can pass a file to `bolt command run`
+or `Invoke-BoltCommand` in order to execute multiline commands.
```

Comment:
```suggestion
Because some devices don't allow you to write files, you might not be able to use `bolt script run` or
`Invoke-BoltScript` to run multiline commands. Instead, you can pass a file to `bolt command run`
or `Invoke-BoltCommand` in order to execute multiline commands.
```